### ISSUE TITLE
cancelled tasks in checkermaster

### ISSUE THREAD

**Issue by siccegge** [fausecteam/ctf-gameserver#7]
Remove cancelled but not yet cleaned tasks from open_tasks in checkermaster

**Comment by siccegge** [comment]
```
ERROR:root:Exception while trying to update service status in database
Traceback (most recent call last):
  File "/usr/bin/ctf-checkermaster", line 167, in get_fresh_jobs
    if task.result() is None:
  File "/usr/lib/python3.4/asyncio/futures.py", line 269, in result
    raise CancelledError
concurrent.futures._base.CancelledError
ERROR:root:Exception while trying to update service status in database
Traceback (most recent call last):
  File "/usr/bin/ctf-checkermaster", line 177, in get_fresh_jobs
    (service, team, tick, result))
psycopg2.IntegrityError: duplicate key value violates unique constraint "scoring_statuscheck_service_id_team_id_tick_key"
DETAIL:  Key (service_id, team_id, tick)=(1, 87, 47) already exists.
```

**Comment by siccegge** [comment]
I proclaim this is solved in 6e1918ab15c6db5999a634cfa01fc8422562617c
